src/analyses/apron/apronAnalysis.apron.ml

```diff
@@ -373,18 +373,18 @@ struct
           invalidate_one ask ctx st (Lval.CilLval.to_lval lval)
         ) rs st
 
-
   let special ctx r f args =
     let ask = Analyses.ask_of_ctx ctx in
     let st = ctx.local in
     let desc = LibraryFunctions.find f in
     match desc.special args, f.vname with
     (* TODO: assert handling from https://github.com/goblint/analyzer/pull/278 *)
-    | `Assert expression -> st
-    | `Unknown "__goblint_check" -> st
-    | `Unknown "__goblint_commit" -> st
-    | `Unknown "__goblint_assert" -> st
-    | `ThreadJoin (id,retvar) ->
+    | Assert expression, _ -> st
+    | Unknown, "__goblint_check" -> st
+    | Unknown, "__goblint_commit" -> st
+    | Unknown, "__goblint_assert" -> st
+    | ThreadJoin { thread = id; ret_var = retvar }, _ ->
+      (* nothing to invalidate as only arguments that have their AddrOf taken may be invalidated *)
```

Comment:
I guess this comment creeped back in with the merge, but was supposed to be removed by this PR.